.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,42 @@
+<!-- 작성 예시는 PR에 표시되지 않습니다. -->
+
+**구현 목표**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)